src/table-view/table-view.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment } from 'react'
+import createReactClass from 'create-react-class'
+import { withStyles, Button } from '@material-ui/core'
+import _ from 'lodash'
+import Select from '../select'
+import Table from '../table'
+
+const TableView = createReactClass({
+  displayName: 'TableView',
+  propTypes: {},
+  getDefaultProps () {
+    return {
+      rows: [],
+      columns: []
+    }
+  },
+  getInitialState () {
+    return {
+      selectedValue: {},
+      selectArray: {},
+      rows: this.props.rows,
+      columns: this.props.columns,
```

Comment:
columns doesn't change, why do you need to save it in state?

Reply:
Yes, I agree. I'll remove that.